README.md

```diff
@@ -1,5 +1,7 @@
 # BuildingSync
 
+![BuildingSync-gem](https://github.com/BuildingSync/BuildingSync-gem/actions/workflows/continuous_integration.yml/badge.svg?branch=develop)
```

Comment:
🆒

Reply:
I think I need to cherry pick this commit. The formatting of the files definitely caused some issues....